### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by apollisa**
### Changes

This is intended to improve the UX across the whole app. This includes:

- Improved wording
- Improved display of services options
- Improved registering requirements

### CODE REVIEW

File: front/src/components/AnimalForm.tsx [hunk 1]
```diff
@@ -50,14 +54,15 @@ export class AnimalForm extends React.Component<AnimalFormProps, AnimalFormState
                             }).catch(() => this.setState({ error: 'Erreur lors de l’enregistrement de l’animal' }));
                         // Else we are editing an existing animal
                         else
-                            AnimalService.update(animal).then(a => {
-                                this.props.onSuccess(a);
+                            AnimalService.update(animal).then(_ => {
+                                console.log(animal);
```
Comment (Hackatosh) [inline front/src/components/AnimalForm.tsx]: Please delete console.log

File: front/src/components/Map.tsx [hunk 2]
```diff
@@ -1,146 +1,170 @@
...
-          ))}
-          </ul>
-
-          </form>
-          <button className="btn btn-primary btn-block" onClick={this.queryAPIServices}>Recherche</button>
-        </div>
-      </div>
-    );
-  }
+					{this.state.markers.map(el => {
```
Comment (Hackatosh) [inline front/src/components/Map.tsx]: Make a separate function to make this more readable, please

File: front/src/components/Map.tsx [hunk 3]
```diff
@@ -1,146 +1,170 @@
...
+		return(
+			<div className="container">
+				<div className="row">
+					<div className="col-10 offset-1 col-md-6" style={{ minHeight: '60vh' }}>
+						<LeafletMap center={this.state.userPosition} zoom={this.props.zoom} style={{height: '100%', width: '100%'}}>
+						<TileLayer
+								attribution='&amp;copy <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+								url="https://cartodb-basemaps-1.global.ssl.fastly.net/light_all/{z}/{x}/{y}.png"
+							/>
+						<Marker position={this.state.userPosition} >
```
Comment (Hackatosh) [inline front/src/components/Map.tsx]: This should be a different color, other than blue (already the color of groomers)

Comment (apollisa) [reply front/src/components/Map.tsx]: I think we should change the icon altogether, not just the colour. @Raklyon any idea?

**Pull request merged**